image/image.go

```diff
@@ -269,3 +336,21 @@ func getItemPath(destination, name string) (itemPath string, err error) {
 
 	return itemPath, nil
 }
+
+func unTarFromFile(tarArchive string, destination string) (err error) {
```

Comment:
Could you move content to UnpackTarImage? I don't see any reason for this private function.

Reply:
done